Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -12,6 +12,7 @@
 )
 from acts.examples.reconstruction import (
     addSeeding,
+    SeedingAlgorithm,
```

Comment:
Is this a related change that needs to be included here?

Reply:
This is for selecting the seeding algorithm (e.g. Default vs Orthogonal) in the job option. It is not really connected to the PR title though. I'd keep it here without opening a different PR for this additional trivial change